earthpy/io.py

```diff
@@ -77,17 +76,37 @@ class EarthlabData(object):
     """
     Data storage and retrieval functionality for Earthlab.
 
+    An object of this class is available upon importing earthpy as
+    ``earthpy.data`` that writes data files to the path:
+    ``~/earth-analytics/data/``.
+
     Parameters
     ----------
     path : string | None
-        The path where data is stored.
+        The path where data is stored. NOTE: this defaults to the directory
+        ``~/earth-analytics/data/``.
+
+    Examples
+    --------
+    List datasets that are available for download, using default object:
+
+        >>> import earthpy
+        >>> earthpy.data
+        Available Datasets: ['california-rim-fire', ...]
+
+    Specify a directory when instantiating a new object:
+
+        >>> import earthpy.io as eio
```

Comment:
@lwasser thanks - I didn't realize we were using the convention `import earthpy as et`. So this should be `et.data.path = "."` instead of `earthpy.data.path = "."`?

Reply:
oooh yes please. `et.data.path`
yes that is the convention we've been using for the past year!! but my gosh we had that discussion forever and ever ago :) thank you